pyroengine/__init__.py

```diff
@@ -5,4 +5,3 @@
 # See the LICENSE file in the root of this repository for complete details.
 
 from .version import __version__
-from pyronearEngine import inference
```

Comment:
You delete the line because it was useless ? Otherwise, we should just change `pyronearEngine ` to `pyroengine` ?

Reply:
Well, not useless but it meant:
`from pyroengine import *` would import everything declared in ̀inference`

I thought that `from pyronear.inference import ...` is a better & cleaner interface.